server/integration_tests/test_data/multisv/query_1/chart_config.json

```diff
@@ -85,197 +85,17 @@
               }
             ],
             "title": "Prevalence of Obesity in Counties of California"
-          },
```

Comment:
this is because the new svs and svpgs that were added to the Poverty topic prevent the PovertyByGender, PovertyByRace topics to be opened up .. maybe can:

1. increase the _MAX_CORRELATION_SVS_PER_TOPIC
2. instead of directly adding the svpg to the Poverty topic, add them as topics